docs/architecture/adr/0025-ts-deprecate-enums.md

```diff
@@ -1,7 +1,8 @@
 ---
 adr: "0025"
 status: Accepted
-date: 2025-06-24
+date: 2025-05-30
+accepted: 2025-06-24
```

Comment:
🎨 Would you mind pulling this and the ADR table change into a separate PR? We'll briefly chat about tracking this. Only as of today have we really had a difference in the dates, and I am also unsure if we _want_ this info.

Reply:
Addressed by cbcbce8c45d0f20a4aa3e0 (revert) and 041223df4bbc42e3e62e20 (restore front matter).

The PR containing the front matter update is #626.